### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by maximeroucher**

### CODE REVIEW

File: coverage/lcov.info
Comment (armanddidierjean) [inline coverage/lcov.info:1]: I think we should ignore this file

**Review by armanddidierjean — COMMENTED** [review]
Could you add a line in the readme explaining how to run the test suite?

**Review by Petitoto — APPROVED** [review]

File: test/admin/admin_test.dart [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+          'SimpleGroup(name: Nom, description: Description, id: )');
+    });
+
+    test('Should parse a group from json', () async {
+      final group = SimpleGroup.fromJson({
+        "id": "1",
+        "name": "name",
+        "description": "description",
+      });
+      expect(group.name, 'name'); // capitaliseAll
```
Comment (armanddidierjean) [inline test/admin/admin_test.dart:50]:
```suggestion
      expect(group.name, 'name');
```

**Review by armanddidierjean — CHANGES_REQUESTED** [review]

File: pubspec.yaml [hunk 2]
```diff
@@ -81,3 +82,22 @@ flutter:
...
+  web:
+    generate: true
+    image_path: "assets/images/icon.png"
+  windows:
+    generate: true
+    image_path: "assets/images/icon.png"
+    icon_size: 48 # min:48, max:256, default: 48
+  macos:
+    generate: true
+    image_path: "assets/images/icon.png"
```
[the remark this answers is not in the document]
Comment (maximeroucher) [reply pubspec.yaml:103]: It might have been included by error

File: test/admin/admin_test.dart [hunk 3]
```diff
@@ -0,0 +1,155 @@
+// ignore_for_file: invalid_use_of_protected_member
```
[the remark this answers is not in the document]
Comment (maximeroucher) [reply test/admin/admin_test.dart:1]: It was causing a linting error, it seems to have been fixed since, I'm removing it

**Review by armanddidierjean — APPROVED** [review]

**Review by armanddidierjean — APPROVED** [review]

**Pull request merged**